cwt-generator failed with error

Generating sources for Fedora `container/python3` by cwt failed with error.

```bash
+ '[' -z /tmp/betka-generator/20200515100741-s2i-python-container ']'
+ '[' -z python3 ']'
+ '[' -z s2i-python-container ']'
+ SOURCES=/tmp/betka-generator/20200515100741-s2i-python-container
+ RESULTS=/tmp/betka-generator/20200515100741-s2i-python-container/results
+ CWT_WD_PREFIX=sync
+ CWT_WD=/tmp/sync
+ CWT_CONFIG_DIR=/tmp/container-workflow-tool/container_workflow_tool/config
+ cd /tmp/betka-generator/20200515100741-s2i-python-container/results
++ git rev-parse --abbrev-ref HEAD
+ config=f31
+ echo f31
+ grep -q master
+ cd /tmp/container-workflow-tool
+ read -r _ ups_name _
++ cwt --config /tmp/container-workflow-tool/container_workflow_tool/config/f31.yaml --do-image python3 utils listupstream
+ mkdir -p /tmp/sync/upstreams/
+ cp -r /tmp/betka-generator/20200515100741-s2i-python-container/s2i-python-container /tmp/sync/upstreams
+ mv /tmp/sync/upstreams/s2i-python-container /tmp/sync/upstreams/python3
+ cp -r /tmp/betka-generator/20200515100741-s2i-python-container/results /tmp/sync
+ mv /tmp/sync/results /tmp/sync/python3
+ cwt -v 5 --config /tmp/container-workflow-tool/container_workflow_tool/config/f31.yaml --disable-klist --base sync --do-image python3 git --commit-msg '' pullupstream
Using working directory: /tmp/sync
Using existing downstream repo: python3
container_workflow_tool.main.dist-git - DEBUG: No release information found in Dockerfile
Using existing repository.
container_workflow_tool.main.dist-git - DEBUG: Running commands in upstream repo.
container_workflow_tool.main.dist-git - DEBUG: Running '1' command 'make generate-all'
container_workflow_tool.main.dist-git - DEBUG: Running '2' command 'for ver in */; do make $ver/root/help.1; done'
container_workflow_tool.main.dist-git - ERROR: b"make: *** No rule to make target 'examples//root/help.1'.  Stop.\nmake: *** No rule to make target 'imagestreams//root/help.1'.  Stop.\nmake: *** No rule to make target 'specs//root/help.1'.  Stop.\nmake: *** No rule to make target 'test//root/help.1'.  Stop.\n"
ERROR: '['for', 'ver', 'in', '*/;', 'do', 'make', '$ver/root/help.1;', 'done']' failed
```

--- a/container_workflow_tool/config/f28.yaml
+++ b/container_workflow_tool/config/f28.yaml
@@ -42,8 +42,7 @@ v1:
   urls: !include share/urls.yaml
 
   commands:
-    2: "for ver in */; do make $ver/root/help.1; done"
-    3: "sed -i 's|registry.fedoraproject.org/f[0-9]*/|registry.fedoraproject.org/f28/|' */Dockerfile.fedora"
+    2: "sed -i 's|registry.fedoraproject.org/f[0-9]*/|registry.fedoraproject.org/f28/|' */Dockerfile.fedora"
 
   # format:
   # - image_name

--- a/container_workflow_tool/config/f29.yaml
+++ b/container_workflow_tool/config/f29.yaml
@@ -42,8 +42,7 @@ v1:
   urls: !include share/urls.yaml
 
   commands:
-    2: "for ver in */; do make $ver/root/help.1; done"
-    3: "sed -i 's|registry.fedoraproject.org/f[0-9]*/|registry.fedoraproject.org/f29/|' */Dockerfile.fedora"
+    2: "sed -i 's|registry.fedoraproject.org/f[0-9]*/|registry.fedoraproject.org/f29/|' */Dockerfile.fedora"
 
   # format:
   # - image_name

--- a/container_workflow_tool/config/f30.yaml
+++ b/container_workflow_tool/config/f30.yaml
@@ -34,7 +34,7 @@ v1:
 ##-5.0      - passenger
 ##-5.28      - perl
 ##-7.2      - php
-##-3.7      - python3
+        - python3
         - ruby
 
   current: *f30
@@ -42,8 +42,7 @@ v1:
   urls: !include share/urls.yaml
 
   commands:
-    2: "make generate"
-    3: "sed -i 's|registry.fedoraproject.org/f[0-9]*/|registry.fedoraproject.org/f30/|' */Dockerfile.fedora"
+    2: "sed -i 's|registry.fedoraproject.org/f[0-9]*/|registry.fedoraproject.org/f30/|' */Dockerfile.fedora"
 
   # format:
   # - image_name

--- a/container_workflow_tool/config/f31.yaml
+++ b/container_workflow_tool/config/f31.yaml
@@ -188,7 +188,7 @@ v1:
       git_branch: "fFEDORA"
       user: "pkubat"
       commands:
-        4: "sed -i 's|registry.fedoraproject.org/fedora:[0-9]*|registry.fedoraproject.org/fedora:29|' */Dockerfile.fedora"
+        4: "sed -i 's|registry.fedoraproject.org/fedora:[0-9]*|registry.fedoraproject.org/fedora:31|' */Dockerfile.fedora"
     toolchain:
       bz_version: "FEDORA"
       component: "toolchain"

--- a/container_workflow_tool/config/f31.yaml
+++ b/container_workflow_tool/config/f31.yaml
@@ -42,8 +42,7 @@ v1:
   urls: !include share/urls.yaml
 
   commands:
-    2: "for ver in */; do make $ver/root/help.1; done"
-    3: "sed -i 's|registry.fedoraproject.org/f[0-9]*/|registry.fedoraproject.org/f29/|' */Dockerfile.fedora"
+    2: "sed -i 's|registry.fedoraproject.org/f[0-9]*/|registry.fedoraproject.org/f31/|' */Dockerfile.fedora"
 
   # format:
   # - image_name

